packages/fscomponents/src/components/CMSBannerStacked.tsx

```diff
@@ -13,9 +13,8 @@ import { CMSProviderProps, withCMSProvider } from './CMSProvider';
 import { CMSBanner, CMSBannerProps } from './CMSBanner';
 
 export interface CMSBannerStackedProps extends CMSBannerProps, CMSProviderProps {
-  carouselProps?: any;
   accessible?: boolean;
-  getAccessibilityLabel?: (instance: any) => string;
+  getAccessibilityLabel?: <T>(instance: T) => string;
```

Comment:
Can you please change this to `<T = any>` so that this won't break existing implementations?